draco/core/io.py

```diff
@@ -982,6 +982,131 @@ def process(self):
         raise pipeline.PipelineStopIteration
 
 
+class ZarrZipHandle:
+    """A handle for keeping track of background Zarr-zipping job."""
+
+    def __init__(self, filename: str, handle: Optional[subprocess.Popen]):
+        self.filename = filename
+        self.handle = handle
+
+
+class SaveZarrZip(ZipZarrContainers):
+    """Save a container as a .zarr.zip file.
+
+    This task saves the output first as a .zarr container, and then starts a background
+    job to start turning it into a zip file. It returns a handle to this job. All these
+    handles should be fed into a `WaitZarrZip` task to ensure the pipeline run does not
+    terminate before they are complete.
+
+    This accepts most parameters that a standard task would for saving, including
+    compression parameter overrides.
+    """
+
+    # This keeps track of the global number of operations run such that we can dispatch
+    # the background jobs to different ranks
+    _operation_counter = 0
+
+    def setup(self):
+        """Check the parameters and determine the ranks to use."""
+
+        if not self.output_name.endswith(".zarr.zip"):
+            raise ConfigError("File output name must end in `.zarr.zip`.")
+
+        # Trim off the .zip suffix and fix the file format
+        self.output_name = self.output_name[:-4]
+        self.output_format = fileformats.Zarr
```

Comment:
This seems to break right now because the `config` property for `output_format` expects a string and tries to call `.lower()` on it. I submitted a PR (radiocosmology/caput#237) to address it, as I think that setting it this way should be valid.